pkg/setting/setting.go

```diff
@@ -410,6 +410,8 @@ type Cfg struct {
 	// LiveAllowedOrigins is a set of origins accepted by Live. If not provided
 	// then Live uses AppURL as the only allowed origin.
 	LiveAllowedOrigins []string
+	// LiveNumberOfRandom60HzChannels number of random 60Hz channels provided by the test datasource
+	LiveNumberOfRandom60HzChannels int
```

Comment:
do we need this from config?  can we just set 10?

and make sure they only start when requested?